webrisk/snippets/compute_threatlist_diff.py

```diff
@@ -70,12 +75,12 @@ def compute_threatlist_diff(
     # https://cloud.google.com/web-risk/docs/reference/rpc/google.cloud.webrisk.v1#computethreatlistdiffresponse
     # Type of response: DIFF/ RESET/ RESPONSE_TYPE_UNSPECIFIED
     print(response.response_type)
-
     # New version token to be used the next time when querying.
     print(response.new_version_token)
-
     # Recommended next diff timestamp.
     print(response.recommended_next_diff)
 
-    print("Obtained threat list diff.")
+    return response
+
+
```

Comment:
Is this newline necessary for some reason?

Reply:
I mean, these two newlines :)

Reply:
'black' auto-formatted to include two new lines. Hence, didn't change it.

Should I go ahead and remove the extra new line for all samples?

Reply:
I wonder if the region tag comment had a blank line after it if black would have added two lines here, or only one. I can't find black rules that explain this specific case.